src/controller/python/test_client/rpc_client.py

```diff
@@ -109,6 +109,22 @@ async def main():
         fabric_id = server_1.get_fabric_id()
         print(f"fabric_id: {fabric_id}")
 
+        # Network Provisioning commands
+        ssid="str:UncharteredTerretory"
```

Comment:
You don't need encoded string here?

Reply:
No this works. The ParseEncodedString method does the formatting based on value in dict